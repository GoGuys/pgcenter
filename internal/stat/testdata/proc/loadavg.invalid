2.43 2.30
