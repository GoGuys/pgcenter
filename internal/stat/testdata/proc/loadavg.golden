2.43 2.30 1.74 1/1697 2277691
